framework/source/class/qx/theme/manager/Decoration.js

```diff
@@ -122,8 +122,11 @@ qx.Class.define("qx.theme.manager.Decoration",
       // create and add a CSS rule
       var css = "";
       var styles = instance.getStyles(true);
-      for (var key in styles) {
-
+      
+      // Sort the styles so that more specific styles come after the group styles, 
+      //  eg background-color comes after background
+      var t = this;
+      Object.keys(styles).sort().forEach(function(key) {
```

Comment:
This bugs me (as does the original unspecified order). The fix just "happens" to work because the two styles we care about happen to be lexicographicaly in the correct order, but who knows what other changes this is making. I would much prefer to see the specific values that we know of that must be in a specific order being explicitly placed in that order. (I agree, though, that using a sorted order to start with is better than the browser-specific (or worse) order provided by "for (keys in styles)".